frank-doc-doclet/src/main/java/org/frankframework/frankdoc/feature/Description.java

```diff
@@ -39,16 +39,31 @@ private Description() {
 	public String valueOf(FrankMethod method) {
 		String result = null;
 		FrankAnnotation annotation = method.getAnnotation(FrankDocletConstants.IBISDOC);
-		if(annotation != null) {
+		if (annotation != null) {
 			try {
 				ParsedIbisDocAnnotation ibisDoc = new ParsedIbisDocAnnotation(annotation);
 				result = ibisDoc.getDescription();
 			} catch(FrankDocException e) {
 				log.error("Could not parse annotation [{}] on method [{}]", FrankDocletConstants.IBISDOC, method.toString());
 			}
 		}
-		if(result == null) {
+		if (result == null) {
 			result = method.getJavaDoc();
```

Comment:
Werkt dit ook op een Class? En op @return, @param en @throws?

Reply:
Een child class inherit de documentatie niet wanneer je een `{@inherhitDoc}` tag gebruikt. Zie het volgende voorbeeld:

```java
/**
 * This is the parent documentation!
 */
public class Parent { }

/**
 * {@inheritDoc}
 */
class Child extends Parent { }
```

Dit is naar mijn mening ook niet heel belangerijk voor de Frankdoc, omdat de documentatie van een parent class al weergeven kan worden met de "Show inheritance" knop.


@return en @throws worden niet ondersteund door de Frankdoc. Ik zal morgen aan Niels vragen of hij het nuttig zou vinden om `@ff.parameter` en `@ff.parameters` te kunnen gebruiken met de `{@inheritDoc}` tag.

Reply:
Ok, prima! Ik las het in een van de oracle links die je erbij had gezet, vandaar!